lib/internal/util.js

```diff
@@ -379,6 +434,7 @@ module.exports = {
   getSystemErrorName,
   getIdentificationOf,
   isError,
+  isInsideNodeModules,
   join,
```

Comment:
Just wanted to say that outside of the box ideas like `isInsideNodeModules` are some of my favorite things.
I really dig it ❤️